packages/react-components/src/components/Icon/Icon.module.scss

```diff
@@ -2,6 +2,10 @@
   align-items: center;
   display: flex;
 
+  svg {
+    pointer-events: none;
+  }
```

Comment:
Can you elaborate on this one?

Reply:
To prevent the list open when the user is clicking on the selected item tag icon (multipicker mode), we need to check the event target. This prop will prevent the event handler to see svg as the target object.